dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/validation/DefaultValidationService.java

```diff
@@ -70,7 +71,13 @@ private ValidationResult validate(TrackerBundle bundle, Validator<TrackerBundle>
           "Skipping validation for metadata import by user '"
               + bundle.getUsername()
               + "'. Not recommended.");
-      return Result.empty();
+      return new Result(
```

Comment:
That was the bug you mentioned right? We did skip validation but also removed all the entities to be persisted 😬

Reply:
Yes, that is it. Skip validation was actually skipping everything